modules/main-rules/pages/add-up-assault-values.adoc

```diff
@@ -1,7 +1,8 @@
 = CC1. Add up Assault values
 
 Both players: Add up the Assault values of all your units that are in base-to-base contact with the enemy.
-A maximum of two units can 'gang up' on each enemy unit.
+
+include::partial$ganging-up.adoc[]
```

Comment:
The partial makes sense if we need to repeat the note, of course -- but looking at it now, I don't think that this is really a logical place to repeat it anyway. Now that we have it in the proper context at `assault-moves-and-charges.adoc`, I don't think we need to follow GW's original odd logic of having it here.
```suggestion
```